### PR TITLE
Rename StaticState into InitialState

### PR DESCRIPTION
**Pull request by alejandraarri**

### CODE REVIEW

File: packages/pci-sdk-iframe/src/hooks/useApplicationState.ts [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 	const { state, dispatch } = usePureState<IApplicationState>({
 		cvv: '•••',
 		exp: '••/••',
 		uiStatus: 'CARD_DATA_HIDDEN',
 		isLoading: false,
 		message: '',
-		pan: `•••• •••• •••• ${staticState.lastFour}`,
-		theme: themes[staticState.theme],
+		pan: `•••• •••• •••• ${initialState.lastFour}`,
+		theme: themes[initialState.theme as IThemeName],
```
Comment (alejandraarri) [inline packages/pci-sdk-iframe/src/hooks/useApplicationState.ts]: I don’t know why TS is now complaining about this 🤷🏻

**Comment by IagoLast** [comment]
I agree that the name is not good, however it seems to me that initialState can be misleading because what is stored in that variable is not the initial value of the react application state. If not a set of "things" communicated from outside to the iframe.

The "put your PIN" message does not belong to the state is something that is injected through the URL and does not change during the entire life cycle of the application.

I think we should look for a more suitable name, in fact I think we should not refer to the state: Staticproperties? Properties? UrlParameters? InyectedOptions? Options? Configuration?

Also, we got Ts Errors :trollface:

**Comment by alejandraarri** [comment]
I was thinking the same, but as we mentioned "initialState" in the mob programming session I've stuck to that 😄  Let's search for a better name, then 👍🏻

**Comment by alejandraarri** [comment]
I have renamed to ConfigOptions. What do you think? 

I can't see the TS errors you say 🤔

File: packages/pci-sdk-iframe/src/hooks/useApplicationState.ts (hunk 1 above)
Comment (IagoLast) [reply packages/pci-sdk-iframe/src/hooks/useApplicationState.ts]: this tsError !

**Pull request merged**